src/core/Kint.ts

```diff
@@ -0,0 +1,219 @@
+import { KintRequest } from "./models/KintRequest";
+import { KintResponse } from "./models/KintResponse";
+import { PostProcessingMiddleware } from "./models/middleware/PostProcessingMiddleware";
+import { PostProcessingMiddlewareTuple } from "./models/middleware/PostProcessingMiddlewareTuple";
+import { PreProcessingMiddlewareTuple } from "./models/middleware/PreProcessingMiddlewareTuple";
+import { PreProcessingMiddleware } from "./models/middleware/PreProcessingMiddleware";
```

Comment:
Incorrect captialisation again. There might but some funky git commands you need to run to get it to recognised a change in captilisation of a file.
```suggestion
import { PreProcessingMiddlewareTuple } from "./models/middleware/PreprocessingMiddlewareTuple";
```